### PR TITLE
Endpoints for integration tests

### LINKED ISSUE #476: Add debug only endpoint to fill rule related tables
For some integrations tests(for example like/dislike endpoints) it's necessary to have a rule in the database.

### LINKED ISSUE #477: Add debug only endpoint to get user's vote on rule
It's necessary for integration tests to be sure like/dislike endpoints work.

### PR DESCRIPTION
**Pull request by Sergey1011010**
# Description

Fixes #476
Fixes #477

## Type of change

Please delete options that are not relevant.

- New feature (non-breaking change which adds functionality)
- This change requires a documentation update

## Testing steps

`make before_commit`

### CODE REVIEW

File: server/server.go [hunk 1]
```diff
@@ -247,55 +247,153 @@ func (server *HTTPServer) checkVotePermissions(writer http.ResponseWriter, reque
...
+
+	err = responses.SendResponse(writer, responses.BuildOkResponseWithData(
+		"rule_error_key", ruleErrorKey,
+	))
 	if err != nil {
 		log.Error().Err(err).Msg(responseDataError)
 	}
 }
 
+func (server *HTTPServer) readVoteOnRuleParams(writer http.ResponseWriter, request *http.Request) (types.ClusterName, types.RuleID, types.UserID, bool) {
```
Comment (bond95) [inline server/server.go:356]: I think we should extract functions like this to separate file, and in `server.go` leave only endpoints

Comment (tisnik) [reply server/server.go:356]: orgId don't have to be handled. It's per-user vote

Comment (bond95) [reply server/server.go:356]: @tisnik in that case how do we check permissions? What if user orgId and cluster orgId don't match?

File: server/server.go [hunk 2]
```diff
@@ -247,55 +247,153 @@ func (server *HTTPServer) checkVotePermissions(writer http.ResponseWriter, reque
...
+		return "", "", "", false
+	}
+
+	_, err = server.Storage.GetRuleByID(ruleID)
+	if err != nil {
+		handleServerError(writer, err)
+		return "", "", "", false
+	}
+
+	err = server.checkVotePermissions(writer, request, clusterID)
```
Comment (bond95) [inline server/server.go:388]: I found it.

File: server/server.go (hunk 1 above)
Comment (bond95) [reply server/server.go:356]: Found it

Comment (Sergey1011010) [reply server/server.go]: done

**Pull request merged**